package.json

```diff
@@ -59,5 +59,15 @@
     "lint:fix": "lerna run lint:fix --parallel --stream",
     "postinstall": "git config core.hooksPath '.husky' && echo 'git hooks configured'",
     "reset:hooks": "git config core.hooksPath '.git' && echo 'git hooks was reset'"
+  },
+  "husky": {
```

Comment:
@sedghi - so that we don't see the issues with formatting differences

Reply:
this doesn't work in husky 8, please see in the cs3d the recent husky pr that was merged